core/src/commonMain/kotlin/com/ustadmobile/core/util/ext/ContentJobItemExt.kt

```diff
@@ -0,0 +1,53 @@
+package com.ustadmobile.core.util.ext
+
+import com.ustadmobile.core.db.JobStatus
+import com.ustadmobile.core.generated.locale.MessageID
+import com.ustadmobile.core.impl.UstadMobileSystemImpl
+import com.ustadmobile.lib.db.entities.ContentJobItem
+
+fun Int.isStatusQueuedOrDownloading() = this >= JobStatus.WAITING_MIN && this < JobStatus.COMPLETE_MIN
```

Comment:
The int extension functions aren't used anywhere else...  Two things we could do:

1) Make the int extension function private
2) Better: move this logic into the extension function itself, e.g.

```
fun ContentJobItem?.isStatusQueuedOrDownloading() = this?.cjiRecursiveStatus?.let {it >= JobStatus.WAITING_MIN && it < JobStatus.COMPLETE_MIN }  ?: false
```